invenio_accounts/config.py

```diff
@@ -31,3 +31,7 @@
 # ACCOUNTS_FORCE_BLUEPRINT_REGISTRATION can be use to override
 # ``register_blueprint`` option.
 ACCOUNTS_REGISTER_BLUEPRINT = None
+
+# Set the secure flag on the session cookie to ensure it is only transmitted
+# over a secure channel.
+SESSION_COOKIE_SECURE = True
```

Comment:
@liamkirsh this default value will not be loaded because of (a) https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/ext.py#L202-L204 it doesn't have `ACCOUNTS_` prefix (b) Flask already sets default value for this option.